mappings/net/minecraft/world/chunk/ChunkStatus.mapping

```diff
@@ -138,17 +140,16 @@ CLASS net/minecraft/class_2806 net/minecraft/world/chunk/ChunkStatus
 		ARG 2 structureManager
 		ARG 3 lightingProvider
 		ARG 5 chunk
+	METHOD method_33732 (Lnet/minecraft/class_2806;Ljava/util/concurrent/Executor;Lnet/minecraft/class_3218;Lnet/minecraft/class_2794;Lnet/minecraft/class_3485;Lnet/minecraft/class_3227;Ljava/util/function/Function;Ljava/util/List;Lnet/minecraft/class_2791;)Ljava/util/concurrent/CompletableFuture;
+		ARG 2 world
 	CLASS class_2807 GenerationTask
 		COMMENT A task called when a chunk needs to be generated.
 		METHOD doWork (Lnet/minecraft/class_2806;Ljava/util/concurrent/Executor;Lnet/minecraft/class_3218;Lnet/minecraft/class_2794;Lnet/minecraft/class_3485;Lnet/minecraft/class_3227;Ljava/util/function/Function;Ljava/util/List;Lnet/minecraft/class_2791;)Ljava/util/concurrent/CompletableFuture;
 			ARG 1 targetStatus
 				COMMENT the status the chunk will be set to after the task is completed
+			ARG 3 world
 	CLASS class_2808 ChunkType
 		COMMENT Specifies the type of a chunk
-		FIELD field_12807 Lnet/minecraft/class_2806$class_2808;
-			COMMENT A chunk which is complete and bound to a world.
-		FIELD field_12808 Lnet/minecraft/class_2806$class_2808;
-			COMMENT A chunk which is incomplete and not loaded to the world yet.
```

Comment:
Would be better to map them from their deobfuscated names and keep the docs (this is an annoying bug in Enigma)